ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -227,7 +229,10 @@ message RevokeUserRightsRequest {
 }
 
 message RevokeUserRightsResponse {
-    // The rights that were actually revoked by the request.
+    // The rights actually revoked by a request.
+    // I.e. if a right was present in the request and is missing from the response it means:
+    // - (typically) the user did not have such right,
+    // - (in a rare race condition case) no such user existed.
```

Comment:
I'd say this is over-specifying things. Please remove. Also: if the user does not exist, then I'd expect the endpoint to raise a `NOT_FOUND` error, wouldn't it?

Reply:
> expect the endpoint to raise a NOT_FOUND error

Yes, that's the usual case is `NOT_FOUND`, but if the timings are right, a user could be deleted in a transaction `A` exactly between user-existence-check and right-delete db calls in a transaction `A`. (for a right-delete db call a non-existing-right-for-an-existing-user and non-existent-user are indistinguishable.)

A contrived scenario why this could be misleading: I attempt to revoke `right-1` on `user-1` and get an empty response. I can now naturally assume `user-1` didn't have `right-1`, while in actuality `right-1` existed throughout `user-1` existence.

Reply:
> I'd say this is over-specifying things.
Ok, removed

Reply:
Thanks for the clarification. I believe that's OK.